code/modules/client/preference_setup/loadout/loadout_fluffitems.dm

```diff
@@ -0,0 +1,840 @@
+
+/datum/gear/fluff/sariU_disk
+	path = /obj/item/weapon/disk/limb/eggnerdltd
+	ckeywhitelist = list("cross_exonar")
+	character_name = list("Sari-U")
+
+/datum/gear/fluff/sariE_disk
+	path = /obj/item/weapon/disk/limb/eggnerdltd
+	ckeywhitelist = list("cross_exonar")
+	character_name = list("Sari-E")
+
+//  D CKEYS
+/datum/gear/fluff/dhaeleena_medal
+	path = /obj/item/clothing/accessory/medal/silver/security/fluff/dhael
+	ckeywhitelist = list("dhaeleena")
+	character_name = list("Dhaeleena M'iar")
+
+/* Item not defined.
+/datum/gear/fluff/elliot_medal
+	path = /obj/item/clothing/accessory/medal/medical
+	ckeywhitelist = list("dickfreedomjohnson")
+	character_name = list("Elliot Richards")
+*/
+
+/datum/gear/fluff/elliot_belt
+	path = /obj/item/weapon/storage/belt/champion
+	ckeywhitelist = list("dickfreedomjohnson")
+	character_name = list("Elliot Richards")
+
+/datum/gear/fluff/drake_box
+	path = /obj/item/weapon/storage/box/fluff/drake
+	ckeywhitelist = list("drakefrostpaw")
+	character_name = list("Drake Frostpaw")
+
+/datum/gear/fluff/theseus_coin
+	path = /obj/item/weapon/coin/diamond
+	ckeywhitelist = list("draycu")
+	character_name = list("Theseus")
+	display_name  = "theseus' diamond coin"
+	description = "An engraved coin made of diamond. On the side for heads is printed the year 2541, along with the letter T. On the side for tails, the letter Y can be seen."
+
+/datum/gear/fluff/yonra_box
+	path = /obj/item/weapon/storage/box/fluff/yonra
+	ckeywhitelist = list("draycu")
+	character_name = list("Schae Yonra")
+
+//  E CKEYS
+/datum/gear/fluff/serkii_box
+	path = /obj/item/clothing/shoes/slippers
+	ckeywhitelist = list("eekasqueak")
+	character_name = list("Serkii Miishy")
+
+/datum/gear/fluff/serkii_skirt
+	path = /obj/item/clothing/under/skirt/fluff/serkii
+	ckeywhitelist = list("eekasqueak")
+	character_name = list("Serkii Miishy")
+
+/datum/gear/fluff/jessie_coat
+	path = /obj/item/clothing/suit/storage/hooded/wintercoat/jessie
+	ckeywhitelist = list("epiccharger")
+	character_name = list("Jessie Mare")
+
+/datum/gear/fluff/verd_medal
+	path = /obj/item/clothing/accessory/medal/bronze_heart
+	ckeywhitelist = list("epigraphzero")
+	character_name = list("Verd Woodrow")
+
+//  F CKEYS
+
+//  G CKEYS
+/datum/gear/fluff/eldi_implant
+	path = /obj/item/weapon/implanter/reagent_generator/eldi
+	ckeywhitelist = list("Gowst")
+	character_name = list("Eldi Moljir")
+
+//  H CKEYS
+/datum/gear/fluff/lauren_medal
+	path = /obj/item/clothing/accessory/medal/conduct
+	ckeywhitelist = list("heroman3003")
+	character_name = list("Lauren Zackson")
+
+/datum/gear/fluff/lauren_string
+	path = /obj/item/clothing/accessory/collar/fluff/goldenstring
+	ckeywhitelist = list("heroman3003")
+	character_name = list("Lauren Zackson")
+
+/datum/gear/fluff/belle_sizegun
+	path = /obj/item/weapon/gun/energy/sizegun
+	ckeywhitelist = list("hottokeeki")
+	character_name = list("Belle Day")
+
+/* //Commenting out, no permit, not allowed
```

Comment:
Expired permits, and items that require permits, should just be straight-up deleted instead of commented out.